diff --git a/applications/luci-app-ipsec-vpnd/po/zh-cn/ipsec.po b/applications/luci-app-ipsec-vpnd/po/zh-cn/ipsec.po
index 7276335..d60943e 100644
--- a/applications/luci-app-ipsec-vpnd/po/zh-cn/ipsec.po
+++ b/applications/luci-app-ipsec-vpnd/po/zh-cn/ipsec.po
@@ -8,13 +8,13 @@ msgid "VPN Client IP"
 msgstr "VPN客户端地址段"
 
 msgid "LAN DHCP reserved started IP addresses with the same subnet mask"
-msgstr "VPN客户端使用独立子网段，默认为 10.10.10.2/24"
+msgstr "VPN客户端使用独立子网段，默认为 10.11.11.2/24"
 
 msgid "VPN Client DNS"
 msgstr "VPN客户端DNS服务器"
 
 msgid "DNS using in VPN tunnel.Set to the router's LAN IP is recommended"
-msgstr "指定VPN客户端的DNS地址。推荐设置为 ipsec0 虚拟接口地址，默认为 10.10.10.1"
+msgstr "指定VPN客户端的DNS地址。推荐设置为 ipsec0 虚拟接口地址，默认为 10.11.11.1"
 
 msgid "Account"
 msgstr "账户"
diff --git a/applications/luci-app-ipsec-vpnd/root/etc/config/ipsec b/applications/luci-app-ipsec-vpnd/root/etc/config/ipsec
index 4cd3f64..eb86912 100644
--- a/applications/luci-app-ipsec-vpnd/root/etc/config/ipsec
+++ b/applications/luci-app-ipsec-vpnd/root/etc/config/ipsec
@@ -1,9 +1,9 @@
 
 config service 'ipsec'
-	option clientdns '10.10.10.1'
+	option clientdns '10.11.11.1'
 	option account 'lean'
 	option secret 'myopenwrt'
 	option enabled '0'
 	option password '12345678'
-	option clientip '10.10.10.2/24'
+	option clientip '10.11.11.2/24'
 
diff --git a/applications/luci-app-ipsec-vpnd/root/etc/uci-defaults/luci-ipsec b/applications/luci-app-ipsec-vpnd/root/etc/uci-defaults/luci-ipsec
index 83b4c23..9674c86 100755
--- a/applications/luci-app-ipsec-vpnd/root/etc/uci-defaults/luci-ipsec
+++ b/applications/luci-app-ipsec-vpnd/root/etc/uci-defaults/luci-ipsec
@@ -14,7 +14,7 @@ uci -q batch <<-EOF >/dev/null
 	set network.VPN=interface
 	set network.VPN.ifname="ipsec0"
 	set network.VPN.proto="static"
-	set network.VPN.ipaddr="10.10.10.1"
+	set network.VPN.ipaddr="10.11.11.1"
 	set network.VPN.netmask="255.255.255.0"
 	
 	commit network
