diff --git a/ipt2socks/Makefile b/ipt2socks/Makefile
index e730159..a0defcb 100644
--- a/ipt2socks/Makefile
+++ b/ipt2socks/Makefile
@@ -10,7 +10,7 @@ PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/zfl9/ipt2socks/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=5279eb1cb7555cf9292423cc9f672dc43e6e214b3411a6df26a6a1cfa59d88b7
+PKG_HASH:=73a2498dc95934c225d358707e7f7d060b5ce81aa45260ada09cbd15207d27d1
 
 PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
